cmd/zt_scenario_helpers_for_test.go

```diff
@@ -133,12 +133,45 @@ func (s scenarioHelper) generateCommonRemoteScenarioForLocal(c *chk.C, dirPath s
 	return
 }
 
-// make 50 blobs with random names
-// 10 of them at the top level
-// 10 of them in sub dir "sub1"
-// 10 of them in sub dir "sub2"
-// 10 of them in deeper sub dir "sub1/sub3/sub5"
-// 10 of them with special characters
+func (scenarioHelper) generateCommonRemoteScenarioForSoftDelete(c *chk.C, containerURL azblob.ContainerURL, prefix string) (string, []azblob.BlockBlobURL, []string) {
+	blobList := make([]azblob.BlockBlobURL, 3)
+	blobNames := make([]string, 3)
+	var listOfTransfers []string
+
+	blobURL1, blobName1 := createNewBlockBlob(c, containerURL, prefix+"top")
```

Comment:
A for loop would save the hassle and allow for generating larger scenarios w/ less hassle.